### PR TITLE
Add Optimism icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/10779

**Popularity metric:**

The Similarweb global rank is 63,167. See https://www.similarweb.com/website/optimism.io.

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I resized, unified the two letters and then took the difference between the circle and the unified letters.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11038,6 +11038,11 @@
             "hex": "0A53E0",
             "source": "https://www.opslevel.com"
         },
+        {
+            "title": "Optimism",
+            "hex": "FF0420",
+            "source": "https://optimism.io"
```
Comment (uncenter) [inline _data/simple-icons.json:11044]: I'm not seeing an SVG available here? Where are you getting it from?

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (CodeShakingSheep) [reply _data/simple-icons.json:11044]: I just used your suggested svg from https://github.com/simple-icons/simple-icons/issues/10779#issuecomment-2094949398 Didn't realize that it wasn't from the official website. So, I should put the GitHub repo as source?

Comment (uncenter) [reply _data/simple-icons.json:11044]: See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines.

Comment (CodeShakingSheep) [reply _data/simple-icons.json]: Ok, I updated the source to https://github.com/ethereum-optimism/brand-kit/blob/main/assets/svg/Profile-Logo.svg.

**Review by adamrusted — APPROVED** [review]

**Comment by sshmaxime** [comment]
Thanks @CodeShakingSheep 🙏

**Pull request merged**